src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -106,6 +108,21 @@ class AutomaticSpeechRecognitionPipeline(ChunkPipeline):
     The input can be either a raw waveform or a audio file. In case of the audio file, ffmpeg should be installed for
     to support multiple audio formats
 
+    Example:
+
+    ```python
+    from transformers import pipeline
+
+    pipe = pipeline(model="openai/whisper-base")
+    result = pipe("https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/1.flac")
+
+    assert result == {
+        "text": " He hoped there would be stew for dinner, turnips and carrots and bruised potatoes and fat mutton pieces to be ladled out in thick, peppered flour fat and sauce."
+    }
+    ```
+
+    [More complex examples](pipeline_tutorial)
```

Comment:
```suggestion
    [More complex examples](../pipeline_tutorial)
```
Need to go down one level as this will be in the subfolder `main_classes`